src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -14,6 +14,9 @@ public class ListCommand extends Command {
 
     public static final String MESSAGE_SUCCESS = "Listed all persons";
 
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Shows a list of all persons in the address book. \n"
+            + "Parameters: list\n"
+            + "Example: list";
```

Comment:
Riding on the previous comment, one alternative I'd suggest is to omit the parameters and example and leave it as just the command word and purpose.

Reply:
But already existing MESSAGE_USAGE (ex. message usage for addCommand)  follows this format and can't edit them since those message usages are used not only for this help message but also the error message for each command. I think it would be more proper to have the same format for every message usage. Also, we can't edit existing message usage as they are the specific guidelines for users when they type wrongly.